### PR TITLE
Updated placeholder text

### PR DESCRIPTION
**Pull request by kirbles19**
## Contributing to Cortex XSOAR Content
Make sure to register your contribution by filling the [contribution registration form](https://forms.gle/XDfxU4E61ZwEESSMA)

**The Pull Request will be reviewed only after the contribution registration form is filled.**

## Status
- [x] In Progress
- [ ] Ready
- [ ] In Hold - (Reason for hold)

## Related Issues
fixes: link to the issue

## Description
A few sentences describing the overall goals of the pull request's commits.

## Screenshots
Paste here any images that will help the reviewer

## Minimum version of Demisto
- [ ] 5.0.0
- [ ] 5.5.0
- [ ] 6.0.0

## Does it break backward compatibility?
   - [ ] Yes
       - Further details:
   - [ ] No

## Must have
- [ ] Tests
- [ ] Documentation

### CODE REVIEW

**Comment by fvigo** [comment]
@kirbles19 the build fails, it seems it doesn't like the `[` and `]` in the descriptions. @ShahafBenYakir what do you think?

```
Validating scheme for Packs/StarterPack/Scripts/BaseScript/BaseScript.yml
ERROR:pykwalify.core:validation.invalid
ERROR:pykwalify.core: --- All found errors ---
ERROR:pykwalify.core:["Value '['Enter a description of the argument', 'including any important information users need to know', 'for example', 'default values.']' is not of type 'str'. Path: '/args/0/description'", "Value '['Enter a description of the script', 'including what function it performs and any important information users need to know', 'for example required permissions.']' is not of type 'str'. Path: '/comment'", "Value '['Enter a description of the data returned in this output.']' is not of type 'str'. Path: '/outputs/0/description'"]
Packs/StarterPack/Scripts/BaseScript/BaseScript.yml: [ST110] - in <SchemaError: error code 2: Schema validation failed:
 - Value '['Enter a description of the argument', 'including any important information users need to know', 'for example', 'default values.']' is not of type 'str'. Path: '/args/0/description'.
 - Value '['Enter a description of the script', 'including what function it performs and any important information users need to know', 'for example required permissions.']' is not of type 'str'. Path: '/comment'.
 - Value '['Enter a description of the data returned in this output.']' is not of type 'str'. Path: '/outputs/0/description'.: Path: '/'>

```

File: Packs/StarterPack/Scripts/BaseScript/BaseScript.yml [hunk 1]
```diff
@@ -1,19 +1,19 @@
 args:
 - default: false
-  description: Base Script dummy argument.
+  description: [Enter a description of the argument, including any important information users need to know, for example, default values.]
```
Comment (ShahafBenYakir) [inline Packs/StarterPack/Scripts/BaseScript/BaseScript.yml:3]:
```suggestion
  description: '[Enter a description of the argument, including any important information users need to know, for example, default values.]'
```

File: Packs/StarterPack/Scripts/BaseScript/BaseScript.yml [hunk 2]
```diff
@@ -1,19 +1,19 @@
...
   isArray: false
   name: dummy
   required: true
   secret: false
-comment: Base Script for development.
+comment: [Enter a description of the script, including what function it performs and any important information users need to know, for example required permissions.]
```
Comment (ShahafBenYakir) [inline Packs/StarterPack/Scripts/BaseScript/BaseScript.yml:8]:
```suggestion
comment: '[Enter a description of the script, including what function it performs and any important information users need to know, for example required permissions.]'
```

File: Packs/StarterPack/Scripts/BaseScript/BaseScript.yml [hunk 3]
```diff
@@ -1,19 +1,19 @@
...
 commonfields:
   id: BaseScript
   version: -1
 enabled: false
 name: BaseScript
 outputs:
   - contextPath: BaseScript.Output
-    description: Base Script dummy output.
+    description: [Enter a description of the data returned in this output.]
```
Comment (ShahafBenYakir) [inline Packs/StarterPack/Scripts/BaseScript/BaseScript.yml:16]:
```suggestion
    description: '[Enter a description of the data returned in this output.]'
```

**Review by fvigo — APPROVED** [review]

**Pull request merged**